src/containers/webuild-day/TheProgram/Tab.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+export const Tab = ({ index, label, selected, onClick }) => {
+  return (
+    <button
+      tabIndex={selected ? '0' : '-1'}
+      key={`tab-${index}`}
+      onClick={() => onClick(index)}
+      className={`block mr-12 focus:outline-none ${
+        selected ? 'border-b-2 font-bold border-white' : ''
+      }`}
```

Comment:
Should use `text-subprimary` for inactive tab:

![image](https://user-images.githubusercontent.com/12707960/100842110-ec641a00-34aa-11eb-8b8c-ab3b76af4d29.png)